authors/apps/articles/admin.py

```diff
@@ -0,0 +1,5 @@
+from .models import ReportedArticle
+from django.contrib import admin
+
```

Comment:
Hey @CryceTruly i see you are adding the ReportedArticle model to the admin area of they app why is that necessary??

Reply:
@wasibani-roy, the whole idea is, when we report, we report to systems administrators and they can access these reported articles in the admin site. That's why i registered the model in the admin.py